### ISSUE TITLE
Unexpected result when using OpenAPI server templating

### ISSUE THREAD

**Issue by mikemadisonweb** [mrin9/RapiDoc#88]
Seems like OpenAPI server templating is not supported?
Server part of the specification:
```
servers:
  - url: '{schema}://domain-from-spec.com/{basePath}'
    variables:
      basePath:
        enum:
          - v1
        default: v1
      schema:
        enum:
          - http
        default: http
```
Expected:
http://domain-from-spec.com/v1
Actual:
https://ui-domain.com{schema}://domain-from-spec.com/{basePath}

Are there any plans to implement server templating support?

**Comment by mrin9** [comment]
thanks for bringing this up, it was completely overlooked. will implement this in our next patch

**Comment by mrin9** [comment]
this is addressed in commit 729549f1a41f1d6d32501b1d125289faf9126038
you can test it out here https://mrin9.github.io/RapiDoc/examples/server-vars.html

you can either build it from master or wait for the next patch release
